com.unity.netcode.gameobjects/Editor/NetworkBehaviourEditor.cs

```diff
@@ -58,20 +58,15 @@ private void RenderNetworkVariable(int index)
             if (value == null)
             {
                 var fieldType = m_NetworkVariableFields[m_NetworkVariableNames[index]].FieldType;
-                var networkVariable = (INetworkVariable)Activator.CreateInstance(fieldType, true);
+                var networkVariable = (NetworkVariableBase)Activator.CreateInstance(fieldType, true);
                 m_NetworkVariableFields[m_NetworkVariableNames[index]].SetValue(target, networkVariable);
             }
 
             var type = m_NetworkVariableFields[m_NetworkVariableNames[index]].GetValue(target).GetType();
             var genericType = type.GetGenericArguments()[0];
 
             EditorGUILayout.BeginHorizontal();
-            if (genericType == typeof(string))
-            {
-                var networkVariable = (NetworkVariable<string>)m_NetworkVariableFields[m_NetworkVariableNames[index]].GetValue(target);
-                networkVariable.Value = EditorGUILayout.TextField(m_NetworkVariableNames[index], networkVariable.Value);
-            }
```

Comment:
remove because there can't be a string anymore, right? (`unmanaged` constraint)

Reply:
Nicely spotted